ethminer/MinerAux.h

```diff
@@ -815,15 +900,16 @@ class MinerCLI
 
 		PoolManager mgr(client, f, m_minerType);
 		mgr.setReconnectTries(m_maxFarmRetries);
-		mgr.addConnection(m_farmURL, m_port, m_user, m_pass);
-		if (!m_farmFailOverURL.empty()) {
-			if (!m_fuser.empty())
-				mgr.addConnection(m_farmFailOverURL, m_fport, m_fuser, m_fpass);
-			else
-				mgr.addConnection(m_farmFailOverURL, m_fport, m_user, m_pass);
+		mgr.addConnection(m_endpoints[k_primary_ep_ix]);
+		if (!m_endpoints[k_secondary_ep_ix].Host().empty())
+		{
+			if (!m_endpoints[k_secondary_ep_ix].User().empty()) {
```

Comment:
Hm is this still needed? its at least deprecated. Normally each pool connection should have its own username and password.

Reply:
I think, the legacy code allowed for the same user password to be used for both primary and backup, at lest that's how I interpreted the existing code. I replicated it in case some are using it that way.